src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/ControlTemplate.cs

```diff
@@ -17,6 +17,7 @@
 using System.Windows.Navigation;        // PageFunctionBase
 using System.ComponentModel;            // DesignerSerializationVisibilityAttribute & DefaultValue
 using System.Windows.Markup;     // DependsOnAttribute
+using System.Runtime.CompilerServices;
```

Comment:
I am a bit skeptical about importing `CompilerServices` and using `CallerArgumentException` just to know the name of parameter unless you have reasoning around it.

Reply:
Okay, I gave it another thought and seems fine to me.